tap_pipedrive/schemas/recents/dynamic_typing/deals.json

```diff
@@ -183,6 +183,15 @@
     },
     "person_hidden": {
       "type": ["null", "boolean"]
+    },
```

Comment:
Determined the type of the fields from the docs:

- https://developers.pipedrive.com/changelog/post/11-deals-endpoints-to-return-two-new-fields
- https://developers.pipedrive.com/changelog/post/new-field-in-deals-object